natewong1313/stafford-junction
# SJ-137-childCareWaiverBackend

### PR Instructions  

**Prerequisites**:  
- Ensure the following tables exist in your `localhost` database on PhpMyAdmin:  
  - `dbChildren`  
  - `dbUnallowedFoods`  
  - `dbChildCareWaiver` (updated sql table structure per this PR).  
- Import the necessary SQL files or verify the table structures match those listed above. 

**Testing the Form**:  
1. Sign in as a family member.
2. Ensure at lease one child already exist within the family account, if not add one with the 'add child' function.
3. Navigate to the child care waiver form.
4. Use the dropdown menu to select a child at the top of the form.  
5. Complete the form fields with sample data.  
6. Submit the form and confirm:  
   - Data is correctly inserted into the `dbUnallowedFoods`, `dbChildCareWaiver`, and `dbChildren` tables.  
   - The child is no longer in the dropdown box after form submission.
